libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -97,7 +97,7 @@ bool ContractLevelChecker::check(ContractDefinition const& _contract)
 	checkPayableFallbackWithoutReceive(_contract);
 	checkStorageSize(_contract);
 
-	return Error::containsOnlyWarnings(m_errorReporter.errors());
+	return Error::containsAtWorseWarnings(m_errorReporter.errors());
```

Comment:
I'm no native English speaker, but wouldn't it be "at worst" / `containsAtWorstWarnings`?

Reply:
I also had the same question. After searching a bit it seemed `at worse` is the correct one.

Reply:
Can you provide a link that shows the usage of "at worse"? I cannot find it in a dictionary.

Reply:
Why not call it `doesNotContainErrors`? :)

Reply:
Probably better ;)